basics_user/getting-started.md

```diff
@@ -56,15 +56,22 @@ Qubes VM Manager and Command Line Tools
 ---------------------------------------
 
 All aspects of the Qubes system can be controlled using command line tools run under a dom0 console. 
-To open a console window in dom0, either go to Start-\>System Tools-\>Konsole or press Alt-F2 and type `konsole`.
+Opening a console window in dom0 can be done in several ways:
+
+* Go to the Start Menu and click Terminal Emulator
+* Press Alt-F3, type `xfce terminal` and press Enter twice
```

Comment:
I don't have the default keybindings, so I'm not sure about the `Alt+F3` part, but when I use the Application Finder in 3.2, I have to type `xfce4-terminal`. `xfce terminal` just returns a red "stop sign" icon and doesn't launch anything.

Reply:
It's the other Application Finder mode, the one you're talking about is "collapsed" (just one line), but the Alt+F3 is by default full one. Not sure about your keybindings, but you can get the full one by pressing down arrow in the collapsed one.

Reply:
Ah, yes, I see what you mean. In the "full" version, searching for `xfce terminal` works as described.